templates/vsphere/cluster-template.yaml

```diff
@@ -0,0 +1,1457 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  labels:
+    cluster.x-k8s.io/cluster-name: ${CLUSTER_NAME}
+  name: ${CLUSTER_NAME}
+  namespace: default
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+        - 192.168.0.0/16
```

Comment:
let's use the same CIDRs we have in other places, `10.1.0.0/16` for pods and `10.152.183.0/24` for services